pkg/providers/vsphere/config/ethtool_daemonset.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: ethtool
+  namespace: {{.eksaSystemNamespace}}
+spec:
+  selector:
+    matchLabels:
+      name: ethtool
+  template:
+    metadata:
+      labels:
+        name: ethtool
+    spec:
+      containers:
+        - name: ethtool
+          image: {{.kindNodeImage}}
+          command: ["/bin/sh"]
+          args: ["-c", "ethtool -K eth0 tx-udp_tnl-segmentation off && ethtool -K eth0 tx-udp_tnl-csum-segmentation off && echo 'done' && sleep infinity"]
+          imagePullPolicy: IfNotPresent
+          securityContext:
+            privileged: true
+      hostNetwork: true
+      restartPolicy: Always
+      tolerations:
```

Comment:
do we need tolerations for CP?

Reply:
No i think we would need this. We want to run this on all nodes that exist, so we shouldn't need anything specific for CP

Reply:
Don't the CP nodes have taints? Let's sync offline, I might be missing something